### ISSUE TITLE
Option to suppress/knowl-ize objectives

### ISSUE THREAD

**Issue by mitchkeller** [rbeezer/mathbook#408]
Default or opt-in method to prevent the output of objectives (or hide them in a knowl).  See the thread at https://groups.google.com/d/msg/mathbook-xml-support/X1dLnVC_-WY/TA7aDGa-AwAJ

**Comment by rbeezer** [comment]
Basic implementation at e30713bc, this will be an add-on once that settles down.

**Comment by rbeezer** [comment]
Opt-in knowlization implemented at 51b07e3.  Perhaps the easiest feature request ever at 3 lines.

Switch is
```
html.knowl.objectives = yes|no
```

I'm going to leave eliminating these entirely as a job for custom XSL as a modification of source, not a job for conversion.  So in a thin layer you can do:
```xslt
<xsl:template match="objectives" />
<xsl:template match="objectives" mode="xref-knowl" />
```
No harm in making the knowl, but maybe you would want to keep the `objectives` top-secret?  ;-)

I should write-up this sort of thing more carefully:  #604
